pkg/karmadactl/get/get.go

```diff
@@ -250,11 +265,15 @@ func (g *CommandGetOptions) Complete(f util.Factory) error {
 		return err
 	}
 	g.karmadaClient = karmadaClient
-	return nil
+
+	return g.handleClusterScopeFlags()
 }
 
 // Validate checks the set of flags provided by the user.
 func (g *CommandGetOptions) Validate(cmd *cobra.Command) error {
+	if len(g.Clusters) == 0 && !g.AllClusters && g.IgnoreKarmada {
+		return fmt.Errorf("please specify at least one target cluster")
+	}
```

Comment:
https://github.com/karmada-io/karmada/blob/8a86cd02af528ef3cec56025923b35830751168f/pkg/karmadactl/get/get.go#L268-L274
If `o.AllClusters` is true, can we skip this validation check?

Reply:
fixed, thanks